README.md

```diff
@@ -1,192 +1,125 @@
-       cd  docker-compose
-       docker-compose up
-
-Note that for the "listworkflows" example you need to have the Elasticsearch feature
-enabled on the Temporal Server side. To do this you can run locally with:
-
-       git clone https://github.com/temporalio/docker-compose.git
-       cd  docker-compose
-       docker-compose -f docker-compose-cass-es.yml up
-
-Alternatively you could install the Temporal Server on Kubernetes / Minicube using the [Temporal Helm charts](https://github.com/temporalio/helm-charts).
-Note that in this case you should use the [Temporal CLI (tctl)](https://docs.temporal.io/tctl-v1) tool to create a namespace called "default":
-
-       tctl --ns default n re
-
-## Temporal Web UI
-
-The Temporal Server running in a docker container includes a Web UI, exposed by default on port 8080 of the docker host.
-If you are running Docker on your host, you can connect to the WebUI running using a browser and opening the following URI:
-
-[http://localhost:8080](http://localhost:8080)
-
-If you are running Docker on a different host (e.g.: a virtual machine), then modify the URI accordingly by specifying the correct host and the correct port.
+## Running Samples:
 
-[http://${DOCKER_HOST}:${WEBUI_PORT}](http://${DOCKER_HOST}:${WEBUI_PORT}).
+You can run both "Core" and "SpringBoot" samples from the main samples project directory.
+Details on how to run each sample can be found in following two sections.
+To skip to SpringBoot samples click [here](#Running-SpringBoot-Samples).
 
-If you have deployed the Temporal Server on Kubernetes using Helm Charts, you can use the kubectl command-line tool
-to forward your local machine ports to the Temporal Web UI:
-
-        kubectl port-forward services/temporaltest-web 8088:8088
-        kubectl port-forward services/temporaltest-frontend-headless 7233:7233
-
-With this you should be able to access the Temporal Web UI with [http://localhost:8088](http://localhost:8088).
-
-## Running samples
-
-By default, samples assume relevant Temporal container ports are listening on `localhost`, on port `7233`.
-If this is not the case, you can tunnel traffic to this port to a different one (even running on a different host), using for example `netcat`, `socat`, or `ssh`.
-
-If, for example, you're running Docker on a virtual machine `vm`, you can use `ssh` to tunnel the traffic using the following command before running the samples:
-
-```
-ssh -g -L 7233:localhost:7233 -N user@vm
-```
-
-## Samples directory
-
-The following section lists all available samples.
-Click on the sample link to view the README, which contains instructions on how to run them.
-
-Each sample has an associated unit test which demonstrates the use of the Temporal Java SDK testing API.
-All tests are available under [src/test/java](./src/test/java/io/temporal/samples)
+### Running "Core" samples
+See the README.md file in each main sample directory for cut/paste Gradle command to run specific example.
```

Comment:
There needs to be somewhere where we tell them to run a local server. We don't have to say how (e.g. the docker compose stuff you deleted), but we should link docs or something.

Reply:
done